### PR TITLE
LSP: profile sample updated

### PR DESCRIPTION
**Pull request by anibmurthy**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: local-source-proxy/install.hbs.md [hunk 1]
```diff
@@ -28,7 +28,10 @@ local_source_proxy:
 Where:
 
 - `local_source_proxy.repository` is required. This is the repository where all your source code
-  will be uploaded.
+  will be uploaded. Examples:
+  - Harbor has the form `my-harbor.io/my-project/local-source`.
+  - Docker Hub has the form `my-dockerhub-user/local-source` or `index.docker.io/my-user/local-source`.
+  - Google Cloud Registry has the form `gcr.io/my-project/local-source`.
```
Comment (atmandhol) [inline local-source-proxy/install.hbs.md:34]: Lets add Google Artifact Registry here as well.

- Google Artifact Registry has the form `region-docker.pkg.dev/my-project/local-source/image`

File: install-online/profile.hbs.md [hunk 2]
```diff
@@ -350,6 +360,9 @@ Images are written to `SERVER-NAME/REPO-NAME/workload-name`. Examples:
     - Harbor has the form `repository: "my-project/supply-chain"`.
     - Docker Hub has the form `repository: "my-dockerhub-user"`.
     - Google Cloud Registry has the form `repository: "my-project/supply-chain"`.
+- `EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE` is where local source artifacts are stored in the registry when `--source-image` is not provided during workload creation via Tanzu CLI. This is represented as <registry-server>/<repository-path>. For more information, see [Local Source Proxy](../local-source-proxy/install.hbs.md)
```
Comment (atmandhol) [inline install-online/profile.hbs.md:363]: Should we simplify this to not mention `--source-image` and just say:

`EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE` is where developer's local source is uploaded during inner-loop iterative development using Local source proxy during workload creation via Tanzu CLI.

Comment (anibmurthy) [reply install-online/profile.hbs.md:363]: I would go with a slight modified verbiage as:
`EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE` is where developer's local source is uploaded when using Local source proxy for workload creation via Tanzu CLI.

**Review by atmandhol — APPROVED** [review]

**Comment by atmandhol** [comment]
cc @RichardJJG . Please review and merge. Thanks!

File: local-source-proxy/install.hbs.md [hunk 3]
```diff
@@ -28,7 +28,11 @@ local_source_proxy:
 Where:
 
 - `local_source_proxy.repository` is required. This is the repository where all your source code
-  will be uploaded.
+  will be uploaded. Examples:
+  - Harbor has the form `my-harbor.io/my-project/local-source`.
+  - Docker Hub has the form `my-dockerhub-user/local-source` or `index.docker.io/my-user/local-source`.
+  - Google Artifact Registry has the form `MY-REGISTRY-REGION-docker.pkg.dev/my-project/local-source/image`.
+  - Google Cloud Registry has the form `gcr.io/my-project/local-source`.
```
Comment (atmandhol) [inline local-source-proxy/install.hbs.md:35]: @anibmurthy Lets add ECR here as well:
- AWS ECR has the form `arn:aws:iam::123456789012:role/local-source` and make a note that local-source repo should exist

File: install-online/profile.hbs.md [hunk 4]
```diff
@@ -291,6 +291,16 @@ buildservice:
...
     namespace: "KP-DEFAULT-REPO-SECRET-NAMESPACE"
 
+local_source_proxy:
+  # Takes the value from the `project_path` under `image_registry` section of `shared` by default, but can be overridden by setting a different value.
+  repository: "EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE"
+  push_secret:
+    # Takes the value from the `secret` section under `image_registry` of `shared` by default, but can be overridden by setting a different value.
+    name: "EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE-SECRET"
+    namespace: "EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE-SECRET-NAMESPACE"
+    # When set to `true`, the secret mentioned in this section becomes exportable. This is achieved by automatically creating the necessary SecretExport in the specified namespace.
```
[the remark this answers is not in the document]
Comment (RichardJJG) [reply install-online/profile.hbs.md]: I'll do so

**Pull request merged**